src/main/java/com/coyoapp/tinytask/web/TaskController.java

```diff
@@ -25,17 +29,18 @@
 class TaskController {
 
   private final TaskService taskService;
-
+  @Autowired
+  private TaskRepository taskRepository;
```

Comment:
Also just a hint as this is backend code: Adding this breaks the backend tests, you would have to adapt them.